systems/overview/12-Glossary.md

```diff
@@ -115,26 +115,26 @@ A watchlist is a page which allows any logged-in user to maintain a list of "wat
 
 Wikibase is the software behind [Wikidata](#wikidata). It consists of a set of [extensions](#mediawiki-extension) to the [MediaWiki](#mediawiki) software. These extensions allow Wikidata to manage and display data in [Items](#item) and [Properties](#property).
 
-## [WikibaseClient](https://www.mediawiki.org/wiki/Extension:Wikibase_Client)
+## Wikibase Repository
+
+Wikibase Repository is an application that provides structured [Entity](#entity) data. Its core functionality is implemented as a [MediaWiki extension](#mediawiki-extension) called [WikibaseRepo](https://www.mediawiki.org/wiki/Extension:Wikibase_Repository). It can optionally contain additional [Wikibase extensions](#wikibase-extension) such as WikibaseLexeme.
+
```

Comment:
Are you moving the Wikibase Repository above the Wikibase Client entry here? They were ordered alphabetically. :-)

Reply:
Aaah they were alphabetical.
As said in other reviews I think it we want to make things like that standard in our docs we should write them down and try to add CI for them.

Personally I think it makes sense to have "Wikibase Client" and "Wikibase Repository" next to each other in the glossary as they are closely related as different systems within the docs.
Having "Wikibase Extension" between the 2 leads to confusing reading IMO.
It would make sense to be able to start at the top of the glossary and read down, starting at the high level and slowly getting lower IMO.

Reply:
Had a call and decided to keep it alphabetical for now.